strawberryfields/decompositions.py

```diff
@@ -387,6 +384,23 @@ def mach_zehnder(m, n, internal_phase, external_phase, nmax):
     m, a symmetric beamsplitter combining modes m and n, an internal phase
     shifter on mode m, and another symmetric beamsplitter combining modes m
     and n.
+
+    The resulting matrix is
+    .. math::
+
+       M = i e^{i \phi_{i}/2} \left[\begin{matrix}\sin \left( \phi_{i}/2 \right) e^{i \phi_{e}} & \cos \left( \phi_{i}/2 \right) \\
+       \cos \left( \phi_{i}/2 \right) e^{i \phi_{e}} & - \sin \left( \phi_{i}/2 \right) \end{matrix}\right]
+
+    Args:
+        m (int): mode number on which the phase shifters act
+        n (int): mode number which is combined with mode m by the beamsplitters
+        internal_phase (float): phase in between the symmetric beamsplitters
+        external_phase (float): phase acting before the first beamsplitter
+        nmax (int): maximum number of modes in the circuit
+
+    Returns:
+        array: unitary matrix of the effective transformation the series of phaseshifters
+            and beamspliiters.
```

Comment:
Note: since `Returns:` can only have one item, line continuations are not indented.

```suggestion
        array: unitary matrix of the effective transformation the series of phaseshifters
        and beamsplitters.
```